x-pack/plugins/apm/scripts/test/e2e.js

```diff
@@ -12,6 +12,7 @@ const yargs = require('yargs');
 const childProcess = require('child_process');
 
 const { argv } = yargs(process.argv.slice(2))
+  .parserConfiguration({ 'unknown-options-as-args': true })
```

Comment:
Do you think it makes sense to not be explicit about the args?

Reply:
Yes. Currently we have to define every single cypress argument ourselves. I don't think that makes sense. Instead we should forward the arguments directly to cypress.